src/Application/ConfigResolver.php

```diff
@@ -46,8 +46,8 @@ final class ConfigResolver
     public static function resolve(array $config, InputInterface $input): Configuration
     {
         $directory = DirectoryResolver::resolve($input);
-        $config = ConfigResolver::mergeInputRequirements($config, $input);
-        $composer = self::getComposer($directory);
+        $config = self::mergeInputRequirements($config, $input);
+        $composer = self::getComposer($directory[0]);
```

Comment:
hmm, so I see you choose that the composer file must be from the first directory which has been added.

I kinda get that, I don't think we will benefit much from allowed multiple composer files. 

The only thing I am worried about is how intuitive this will be for the user.

Reply:
Would could always allow a new parameter, to specify the composer path and if that is not set, then we just fallback to looking at the first file supplied.

Reply:
@olivernybroe Yeah I thought it's simpler to get composer file from first directory (specially when we run analyser without specifying any path). And adding a parameter is nice and I can go for it.